include/pros/rtos.hpp

```diff
@@ -320,6 +321,22 @@ class Task {
 	task_t task;
 };
 
+// STL Clock compliant clock
+class ProsClock {
```

Comment:
per discussion in #127 I'm pretty sure there's a way to add newlib stubs in such a way that some of the std::chrono features Just Work